src/styles/index.scss

```diff
@@ -1,5 +1,7 @@
 @import url('https://fonts.googleapis.com/css?family=Roboto&display=swap');
```

Comment:
Just like the other files, it looks like there are git conflicts on this page. Please resolve all conflicts